src/lib/store/store.types.ts

```diff
@@ -241,4 +241,13 @@ export type StoreManager = {
    * @returns The store manager, to allow chaining.
    */
   parseSingleFileIncludes(fileContent: any): StoreManager;
+
+  /**
+   * Gets data from store or from fileSystem if don't exist
```

Comment:
Better this way: "Gets data from store or from file system if not present in store"